frontend/views/pages/GroupChat.vue

```diff
@@ -204,9 +204,15 @@ export default ({
   }
 }
 
-::v-deep .c-logo {
-  @include touch {
-    display: none;
+::v-deep {
+  .c-logo {
+    @include touch {
+      display: none;
+    }
+  }
+
+  .p-main {
+    height: auto !important;
```

Comment:
The fix made above (`height: fit-content;`) cannot be applied to the `GroupChat.vue` page because, the chat has the setting where message layout fills the entire available height even when the content is not long enough. So applying `height: auto;` only to `/group-chat` page here.